package.json

```diff
@@ -5,13 +5,17 @@
   "main": "src/index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "eslint ./src",
+    "lint": "eslint --ext .js,.ts ./src",
     "lint:fix": "npm run lint --fix",
-    "start": "node src/index.js",
+    "prebuild": "rimraf ./build",
+    "build": "tsc",
+    "prestart": "npm run build",
+    "start": "node build/index.js",
+    "start:dev": "nodemon",
```

Comment:
start:dev vs develop
Are both needed? Can we get rid of one?

Reply:
Reading through the docs further, my understanding (newer to nodemon), start:dev is running the uncompiled version locally without the container and develop is using a built container with everything compiled inside.

Likely both can be kept, but wanted to confirm.

Reply:
You are correct